manim/utils/config_utils.py

```diff
@@ -314,6 +322,25 @@ def _parse_cli(arg_list, input=True):
         "--config_file", help="Specify the configuration file",
     )
 
+    parser.add_argument(
+        "-q", "--quiet",
+        action="store_true",
+        help="Quiet mode. Equivalent to -v CRITICAL. It can be overridden by -v and -L",
+    )
+    parser.add_argument(
+        "-v", "--verbose",
+        type=str,
+        help="Verbosity level. Also changes the ffmpeg log level unless the latter is specified",
+        metavar="loglevel",
+        choices=constants.VERBOSE_CHOICES,
+    )
+    parser.add_argument(
+        "-L", "--ffmpeg",
```

Comment:
**Issue** : I think `--ffmpeg` as a flag could be very misleading. Consider changing it,

Reply:
Okey, I didn't have any good ideas but what about something a little longer but more explicit like `--ffmpeg_loglevel` ?
By the way thanks for the review.